cmd/test/test.go

```diff
@@ -165,60 +168,67 @@ func newTestCommand() *cobra.Command {
 			}
 
 			if !runner.Quiet || exitCode != 0 {
-				if runner.Output == OutputAppstudio {
-					// The appstudio format is unknown to Conftest so we handle it ourselves
+				for _, outputAndFormat := range outputFormats {
+					parts := strings.SplitN(outputAndFormat, "=", 2)
```

Comment:
`parts` can be `nil` here if there is no `=` in `outputAndFormat`, perhaps switch on `len(parts)`?

Reply:
I might be wrong, I think parts would be just `[outputAndFormat]` if there was no "=" char present, so `parts[0]` is safe to use.

Reply:
Could still switch on `len(parts)` for extra readability though.

Reply:
@zregvart Yep, that makes sense. I refactored that bit of code.